todo_app/data/session_items.py

```diff
@@ -1,78 +1,171 @@
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
-
-def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-
-def add_item(title):
-    """
-    Adds a new item with the specified title to the session.
-
-    Args:
-        title: The title of the item.
-
-    Returns:
-        item: The saved item.
-    """
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
+    
 
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
+    cards = []
+    trellolists = json.loads(response.text)
+    for trellolist in trellolists:
+        for card in trellolist ["cards"]:
+            item = Item.from_trello_card(card, trellolist)
+            cards.append(item)
+    
+    return cards
 
-    return item
 
 
-def save_item(item):
-    """
-    Updates an existing item in the session. If no existing item matches the ID of the specified item, nothing is saved.
 
-    Args:
-        item: The item to save.
-    """
-    existing_items = get_items()
-    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
+def get_lists():
+    board_id = os.getenv ('BOARD_ID')
+    key = os.getenv ('API_KEY')
+    token = os.getenv ('API_TOKEN')
+    url = f"https://api.trello.com/1/boards/{board_id}/lists"
 
-    session['items'] = updated_items
+    headers = {
+       "Accept": "application/json"
+    }
 
-    return item
+    query = {
+        "key" : key,
+        "token" : token, 
+        "fields" : "id,name"
+    }
 
+    response = requests.get(
+       url,
+       headers=headers,
+       params = query
+    )
 
-def delete_item(item):
-    existing_items = get_items()
-    existing_items.remove(item)
+    return json.loads(response.text)
 
-    session['items'] = existing_items
     
+def post_add(title):
+    key = os.getenv('API_KEY')
+    token = os.getenv('API_TOKEN')
+    url = f"https://api.trello.com/1/cards"
+
+    headers = {
+       "Accept": "application/json"
+    }
+
+    query = {
+        "key" : key,
+        "token" : token, 
+        "idList" : get_lists()[0]["id"],
+        "name": title
+    }
+
+    response = requests.post(
+       url,
+       headers=headers,
+       params=query
+    )
+
+    print(json.dumps(json.loads(response.text), sort_keys=True, indent=4, separators=(",", ": ")))
```

Comment:
I know the Trello docs include this, but is it something that we actually want to log on every add?